package ru.practicum.mainservice.exceptions;public class DataNotFoundException {
}
